build-tools/packages/build-cli/src/commands/check/policy.ts

```diff
@@ -378,12 +379,12 @@ async function runFinalHandlers(
 
 async function readStdin(): Promise<string> {
 	return new Promise((resolve) => {
-		const stdin = process.openStdin();
+		const { stdin } = process;
```

Comment:
I don't know where process.openStdin can from, but it went away, and this works instead using the more recommended import pattern for accessing process.